### PR TITLE
WIP: initial draft of 3 article archetypes

### PR DESCRIPTION
**Pull request by apreshill**
Hi tidyverse team,

Going through this `blogdown` site for some education team projects and thought this addition might make your ["contributing"](https://github.com/tidyverse/tidyverse.org/blob/master/CONTRIBUTING.md) section a bit easier.

I added 3 article archetypes, which you can use via the RStudio "New Post" add-in that could make life easier (I hope!):

screen shot 2019-02-11 at 12 05 31 pm

1. `default.md`- this recapitulates what happens now and preloads the basic post YAML (which is currently populating from the project-level [.Rprofile](https://github.com/tidyverse/tidyverse.org/blob/master/.Rprofile))
     - this could be a plain markdown or an RMarkdown post
1. `default-rmarkdown.md`- this should only be used with an R Markdown post and provides only the setup chunk at the top:
   ```{r setup, include = FALSE}
   library(testthat) # add / replace as needed
   knitr::opts_chunk$set(
     collapse = TRUE, comment = "#>", 
     fig.width = 7, 
     fig.align = 'center',
     fig.asp = 0.618, # 1 / phi
     out.width = "700px"
    )
   ```
1. `package-release.md`- this is the same as the `default-rmarkdown.md` archetype, and should only be used with R Markdown posts, but adds the `usethis::use_tidy_thanks()` code chunk also:

   ```{r echo = FALSE, eval = FALSE}
   # to use, set eval = TRUE
   library(usethis)
   use_tidy_thanks("OWNER/REPO") ## default: interval = since the last release
   use_tidy_thanks("OWNER/REPO", from = "2018-05-01")
   use_tidy_thanks("OWNER/REPO", from = "v1.3.0")
   ```

I'm not sure if this is useful for you all, or if you already rejected the notion of custom archetypes. But if archetypes are something you may want to integrate, we could work on wording of these, and I could revise the CONTRIBUTING section to reflect using the new post add-in for new articles. Let me know what you think.

Alison

### CODE REVIEW

**Comment by batpigandme** [comment]
This is awesome, Alison!

**Comment by apreshill** [comment]
yay, try taking them for a spin locally and lmk how it goes!

File: archetypes/package-release.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+library(testthat) # add / replace as needed
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
```
Comment (batpigandme) [inline archetypes/package-release.md]: @jennybc, do you think we should add a placeholder here for "You can install {insert package here} from CRAN…" plus installation code chunk (with `eval = FALSE`) here?

Comment (jennybc) [reply archetypes/package-release.md]: Yeah. There are several recent-ish posts where a template could be found or at least derived. Note that many other posts *don't* do this 😕

I think every package release post should include a short standard description of what the package does, how to install it from CRAN, and how to attach via `library()`.

The principle is there shouldn't be some mission critical single line of code that we systematically hide and without this implicit knowledge, nothing in the blog post would actually run for a reader.

File: archetypes/default-rmarkdown.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  author: Kelley Bozarth
+# one of: "case studies", "learn", "package", "programming", or "other"
+categories: [Other] 
+---
+
+
+```{r setup, include = FALSE}
+library(testthat) # add / replace as needed
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>",
```
Comment (batpigandme) [inline archetypes/default-rmarkdown.md]: `fig.path = "figs/"` should go in here as well, right?

**Comment by hadley** [comment]
Is it worth finishing this off? (with updates for new structure?)

**Comment by apreshill** [comment]
Yes- I'll update.

**Comment by hadley** [comment]
Fixes #352

**Comment by hadley** [comment]
@apreshill no rush, just wanted to make sure that we shouldn't just close this PR.

**Comment by hadley** [comment]
Closing, since I've now integrated into the new `index.Rmd` in the blog archetype.

**Pull request closed without merging**